_monitoring-your-cluster/pa/index.md

```diff
@@ -8,57 +8,106 @@ redirect_from:
   - /monitoring-plugins/pa/index/
 ---
 
-# Performance analyzer
+# Performance Analyzer
 
-Performance analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
+Performance Analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
```

Comment:
agent or plugin? Plugin is used later.

Reply:
There's a distinction between both within Performance Analyzer itself. Performance Analyzer as a whole is comprised of:

- The PA plugin, the installation files found in OpenSearch core.
- The RCA agent, which runs independent of the plugins settings but requires the plugin.

It's something I want to circle back on for the doc as a whole with @khushbr